api/permissions_api.py

```diff
@@ -117,29 +116,17 @@ class Meta:
 
 class UnitRoleViewSet(viewsets.ModelViewSet):
     serializer_class = UnitRoleSerializer
-    permission_classes = (
-        [permissions.IsAuthenticated & UnitRolePermission]
-        if not settings.TMP_PERMISSIONS_DISABLED
-        else [permissions.AllowAny]
-    )
+    permission_classes = [permissions.IsAuthenticated & UnitRolePermission]
```

Comment:
Aand this is one of those DRF permissions. It will render empty list for non-autenticated requests if `IsAuthenticated` is removed. Shouldn't be that bad but lets keep this as is.